### PR TITLE
Add indestructible Papyrus

### PR DESCRIPTION
**Pull request by sodomite**

### CODE REVIEW

File: mods/ctf/ctf_map/ctf_map_core/nodes.lua [hunk 1]
```diff
@@ -424,6 +424,25 @@ do
...
+		walkable = false,
+		selection_box = {
+			type = "fixed",
+			fixed = {-6 / 16, -0.5, -6 / 16, 6 / 16, 0.5, 6 / 16},
+		},
+		groups = {immortal = 1},
+		sounds = default.node_sound_leaves_defaults(),
+		after_dig_node = function(pos, node, metadata, digger)
+			default.dig_up(pos, node, digger)
+		end,
```
Comment (Panquesito7) [inline mods/ctf/ctf_map/ctf_map_core/nodes.lua:444]: Not needed as there are no more lines after that. 🙂
```suggestion
		end
```

Comment (realOneplustwo) [reply mods/ctf/ctf_map/ctf_map_core/nodes.lua:444]: I thought it was a convention to leave a comma after the last element of a table.

**Review by moreavy — APPROVED** [review]
Working in the `map_maker` mod.

**Review by Panquesito7 — APPROVED** [review]

**Comment by appgurueu** [comment]
How the actual heck can you approve this

File: mods/ctf/ctf_map/ctf_map_core/nodes.lua (hunk 1 above)
Comment (LoneWolfHT) [reply mods/ctf/ctf_map/ctf_map_core/nodes.lua:444]: Yes it is

**Pull request merged**